app/static/pages/component_summary.js

```diff
@@ -0,0 +1,42 @@
+// On loading the component summary page, set up and populate the various sections to be included
+var pending = [];
+
+$(function()
+{
+  // For each section ...
+  $('div.section').each(function()
+  {
+    // Get information about what type of record this section contains
+    var section    = this;
+    var builtform  = $('.builtform', this);
+    var recordType = builtform.data("recordtype");
+    var record     = builtform.data("record");
+    
+    // Get the type form corresponding to the record
+    var schema_url = '';
+    
+    if(recordType === "component") { schema_url = "/json/componentForms/" + record.formId; }
+    else if(recordType === "test") { schema_url = "/json/testForms/" + record.formId; }
+    else if(recordType === "job")  { schema_url = "/json/jobForms/" + record.formId; }

```

Comment:
Prefer backtick notation:
```schema_url = `/json/componentForms/${record.formId}`;```

Reply:
Should I change it everywhere in the code right now?  Pretty sure there's a lot more places where I've used `"" +` type notation ... everywhere where there's an `href` for example.

Reply:
Just for new changes is fine for now I think.  We’ll automate these changes when we introduce linting.